src/app/static/src/app/components/modelOptions/ModelOptions.vue

```diff
@@ -6,7 +6,7 @@
         </div>
         <dynamic-form v-if="!loading && !hasOptionsError"
                       v-model="modelOptions"
-                      submit-text="Validate"
+                      :submit-text="validateText"
```

Comment:
Translate the 'Validate' button in the model options - apparently we weren't doing this for French yet.